web_widget_text_count/static/src/scss/field_text_count.scss

```diff
@@ -0,0 +1,12 @@
+.text-counter-widget{
+        background-color: transparent;
+        float: right;
+        white-space: pre-wrap;
+        border: 1px solid #ccc;
+        border-radius: 3px;
+        padding: 2px 4px;
+        color: #1f1f1f;
+    div.text-counter-widget {
```

Comment:
hmm text-counter-widget inside text-counter-widget? 
You can add your own class if needed (there's a param on widget class for it)